notebook/agentchat_multi_task_async_chats.ipynb

```diff
@@ -27,13 +27,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
     "import autogen\n",
     "\n",
-    "config_list = autogen.config_list_from_json(env_or_file=\"OAI_CONFIG_LIST\")\n",
+    "config_list = autogen.config_list_from_json(env_or_file=\"../OAI_CONFIG_LIST\")\n",
```

Comment:
Please use the original path because we assume the file is under notebook/

Reply:
Fixed